tests/test_drlearner.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) QuantCo 2024-2024
+# SPDX-License-Identifier: BSD-3-Clause
+
+import numpy as np
+import onnxruntime as rt
+import pytest
+from lightgbm import LGBMClassifier, LGBMRegressor
+from onnxmltools import convert_lightgbm, convert_xgboost
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import FloatTensorType
+from sklearn.linear_model import LinearRegression, LogisticRegression
+from sklearn.neighbors import RadiusNeighborsRegressor
+from xgboost import XGBRegressor
+
+from metalearners import DRLearner
+from metalearners._typing import Params
+from metalearners.metalearner import TREATMENT_MODEL
+
+from .conftest import all_sklearn_regressors
+
+
+@pytest.mark.parametrize(
+    "treatment_model_factory, onnx_converter",
+    (
+        list(
+            zip(
+                all_sklearn_regressors,
```

Comment:
Do we have a reason to believe that some of these regressors behave differently than others or is there a guarantee from `convert_sklearn` that we could potentially rely on?

Reply:
I am not sure I understand your question, afaict all these regressor behave in the same way